### PR TITLE
fix(voting): Page overflow fixed

### PR DESCRIPTION
**Pull request by onlychild**
reactmarkdown adopt to <a> Tag

### CODE REVIEW

**Comment by chef-jojo** [comment]
Thanks @onlychild 
I also noticed that Table is causing overflow as well
https://pancakeswap.finance/voting/proposal/Qme3CW5NBYEeECXbFwnNRZME44oBwMRy93fQeRdz7to6qn
Could you help it? I think It'd be great if the table itself can be scrolled

**Comment by onlychild** [comment]
> Thanks @onlychild I also noticed that Table is causing overflow as well https://pancakeswap.finance/voting/proposal/Qme3CW5NBYEeECXbFwnNRZME44oBwMRy93fQeRdz7to6qn Could you help it? I think It'd be great if the table itself can be scrolled

fixed!! by the way，can you granted the develop permission of https://github.com/pancakeswap/pancake-uikit? the ux of the bottomNav Component maybe has some bugs！

**Comment by chef-jojo** [comment]
> > Thanks @onlychild I also noticed that Table is causing overflow as well https://pancakeswap.finance/voting/proposal/Qme3CW5NBYEeECXbFwnNRZME44oBwMRy93fQeRdz7to6qn Could you help it? I think It'd be great if the table itself can be scrolled
> 
> 
> 
> fixed!! by the way，can you granted the develop permission of https://github.com/pancakeswap/pancake-uikit? the ux of the bottomNav Component maybe has some bugs！

Moved here https://github.com/pancakeswap/pancake-toolkit

**Review by chef-jojo — CHANGES_REQUESTED** [review]

File: src/components/ReactMarkdown/styles.tsx [hunk 1]
```diff
@@ -14,6 +14,11 @@ const Table = styled.table`
     padding: 8px;
   }
 `
+const Box = styled.div`
```
[the remark this answers is not in the document]
Comment (onlychild) [reply src/components/ReactMarkdown/styles.tsx]: TableBox

File: src/components/ReactMarkdown/styles.tsx [hunk 2]
```diff
@@ -47,14 +56,22 @@ const markdownComponents: Partial<NormalComponents & SpecialComponents> = {
   p: (props) => {
     return <Text as="p" my="16px" {...props} />
   },
-  table: Table,
+  // table: Table,
```
Comment (chef-jojo) [inline src/components/ReactMarkdown/styles.tsx:59]: please remove this line

Comment (onlychild) [reply src/components/ReactMarkdown/styles.tsx]: done

**Review by chef-jojo — APPROVED** [review]

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**